core/mbed/source/interrupts.cpp

```diff
@@ -26,6 +26,11 @@ void vIRQ_SetVectorX(uint32_t irqn, uint32_t vector, uint32_t flag)
     }
 }
 
+void vIRQ_SetVector(uint32_t irqn, uint32_t vector)
```

Comment:
Ah, yes. That's the whole point of this change.

Is there a reason we can't call gate vIRQ_SetVectorX? or rename vIRQ_SetVectorX to vIRQ_SetVector?